src/utils/injectSaga.js

```diff
@@ -9,40 +9,53 @@ import getInjectors from './sagaInjectors';
  *
  * @param {string} key A key of the saga
  * @param {function} saga A root saga that will be injected
- * @param {string} [mode] By default (constants.RESTART_ON_REMOUNT) the saga will be started on component mount and
- * cancelled with `task.cancel()` on component un-mount for improved performance. Another two options:
- *   - constants.DAEMON—starts the saga on component mount and never cancels it or starts again,
- *   - constants.ONCE_TILL_UNMOUNT—behaves like 'RESTART_ON_REMOUNT' but never runs it again.
+ * @param {string} [mode] By default (constants.DAEMON) the saga will be started
+ * on component mount and never canceled or started again. Another two options:
+ *   - constants.RESTART_ON_REMOUNT — the saga will be started on component mount and
+ *   cancelled with `task.cancel()` on component unmount for improved performance,
+ *   - constants.ONCE_TILL_UNMOUNT — behaves like 'RESTART_ON_REMOUNT' but never runs it again.
  *
  */
 export default ({ key, saga, mode }) => (WrappedComponent) => {
   class InjectSaga extends React.Component {
     static WrappedComponent = WrappedComponent;
-    static displayName = `withSaga(${(WrappedComponent.displayName || WrappedComponent.name || 'Component')})`;
-    static contextTypes = {
-      store: PropTypes.object.isRequired,
-    };
+
+    static contextType = ReactReduxContext;
```

Comment:
why do we need this here?

Reply:
It's a requirement for the latest version of `react-redux`.